### DISCUSSION TITLE
Translation Tool

### DISCUSSION THREAD

**Discussion by oniras65** [maforget/ComicRackCE#17] [General]
Greetings
I did a long search of the tool used for translation, unfortunately the search was unsuccessful.
The cYo site is still accessible via Wayback Machine (archive.org) I was also able to navigate many resources, including the forum. Unfortunately the link of the tool has not been scraped  by the web engine and the information to identify the tool itself is too few. Considering the number of downloads he had (a few dozen) I think the only way to find it, is to try to contact translators, they might still have him in their archives.

In particular sifting through the cache forum I found the reference of Italian translator, I’m trying to contact him to see if he has the tool yet and if interested in participating. 

I wrote him privately, I prefer not to write here his personal data, I await answer.

Greetings

**Comment by Totengeist** [comment]
Did you hear anything back about this?

**Reply by maforget** [reply]
no nothing yet.

**Comment by Ricesvinto** [comment]
Searching on old backups I find Cyo's own translation tool
[localizer.zip](https://github.com/user-attachments/files/24624702/localizer.zip)

**Reply by maforget** [reply]
Thanks for that will take a look

**Comment by maforget** [comment]
Thanks to @Ricesvinto finding the tool it will be much easier to update translations. 

I've made some work on that on PR #235. I will be making a guide on how to create/update languages on the [wiki](https://github.com/maforget/ComicRackCE/wiki/Localization).

To update a language using the tool requires to open a folder and then use the update command with the reference language. It used to be German (de.zip) but it will now be French (fr.zip).
